### ISSUE TITLE
Can´t install systembridgedshared

### ISSUE THREAD

**Issue by rvst1104** [timmo001/system-bridge#1998]
### Description

Hi,

when trying to install system-bridge on my Windows 10 PC with the command provided by the docs

`python -m pip install --upgrade systembridgeshared systembridgebackend systembridgecli systembridgefrontend systembridgegui`

I do get the following error message. 
Any idea what I can do?

`Collecting systembridgeshared
  Downloading systembridgeshared-3.4.7-py3-none-any.whl (26 kB)
Collecting systembridgebackend
  Downloading systembridgebackend-3.4.7-py3-none-any.whl (73 kB)
     ---------------------------------------- 73.5/73.5 kB 118.9 kB/s eta 0:00:00
Collecting systembridgecli
  Downloading systembridgecli-3.4.7-py3-none-any.whl (3.2 kB)
Collecting systembridgefrontend
  Downloading systembridgefrontend-3.4.7-py3-none-any.whl (540 kB)
     ---------------------------------------- 540.3/540.3 kB 69.2 kB/s eta 0:00:00
Collecting systembridgegui
  Downloading systembridgegui-3.4.7-py3-none-any.whl (36 kB)
Collecting incremental==21.3.0
  Downloading incremental-21.3.0-py2.py3-none-any.whl (15 kB)
Collecting colorlog==6.7.0
  Downloading colorlog-6.7.0-py2.py3-none-any.whl (11 kB)
Collecting cryptography==38.0.1
  Downloading cryptography-38.0.1-cp36-abi3-win_amd64.whl (2.4 MB)
     ---------------------------------------- 2.4/2.4 MB 67.0 kB/s eta 0:00:00
Collecting sqlmodel==0.0.8
  Downloading sqlmodel-0.0.8-py3-none-any.whl (21 kB)
Collecting pydantic==1.10.2
  Downloading pydantic-1.10.2-cp37-cp37m-win_amd64.whl (2.1 MB)
     ---------------------------------------- 2.1/2.1 MB 66.3 kB/s eta 0:00:00
Collecting systembridgeshared
  Downloading systembridgeshared-3.4.6-py3-none-any.whl (26 kB)
Collecting pydantic==1.10.1
  Downloading pydantic-1.10.1-cp37-cp37m-win_amd64.whl (2.1 MB)
     ---------------------------------------- 2.1/2.1 MB 68.2 kB/s eta 0:00:00
Collecting systembridgeshared
  Downloading systembridgeshared-3.4.5-py3-none-any.whl (26 kB)
  Downloading systembridgeshared-3.4.4-py3-none-any.whl (26 kB)
Collecting colorlog==6.6.0
  Downloading colorlog-6.6.0-py2.py3-none-any.whl (11 kB)
Collecting pydantic==1.9.1
  Downloading pydantic-1.9.1-cp37-cp37m-win_amd64.whl (1.9 MB)
     ---------------------------------------- 1.9/1.9 MB 69.1 kB/s eta 0:00:00
Collecting appdirs==1.4.4
  Downloading appdirs-1.4.4-py2.py3-none-any.whl (9.6 kB)
Collecting cryptography==37.0.4
  Downloading cryptography-37.0.4-cp36-abi3-win_amd64.whl (2.4 MB)
     ---------------------------------------- 2.4/2.4 MB 63.5 kB/s eta 0:00:00
Collecting aiohttp==3.8.1
  Downloading aiohttp-3.8.1-cp37-cp37m-win_amd64.whl (551 kB)
     ---------------------------------------- 551.8/551.8 kB 150.7 kB/s eta 0:00:00
Collecting systembridgeshared
  Downloading systembridgeshared-3.4.3-py3-none-any.whl (26 kB)
  Downloading systembridgeshared-3.4.2-py3-none-any.whl (26 kB)
  Downloading systembridgeshared-3.4.1-py3-none-any.whl (25 kB)
  Downloading systembridgeshared-3.4.0-py3-none-any.whl (25 kB)
  Downloading systembridgeshared-3.3.4-py3-none-any.whl (25 kB)
  Downloading systembridgeshared-3.3.3-py3-none-any.whl (25 kB)
  Downloading systembridgeshared-3.3.2-py3-none-any.whl (25 kB)
  Downloading systembridgeshared-3.3.1-py3-none-any.whl (25 kB)
  Downloading systembridgeshared-3.3.0-py3-none-any.whl (22 kB)
Collecting cryptography==37.0.2
  Downloading cryptography-37.0.2-cp36-abi3-win_amd64.whl (2.4 MB)
     ---------------------------------------- 2.4/2.4 MB 178.5 kB/s eta 0:00:00
Collecting systembridgeshared
  Downloading systembridgeshared-3.2.1-py3-none-any.whl (21 kB)
  Downloading systembridgeshared-3.2.0-py3-none-any.whl (21 kB)
  Downloading systembridgeshared-3.1.5-py3-none-any.whl (21 kB)
  Downloading systembridgeshared-3.1.4-py3-none-any.whl (21 kB)
  Downloading systembridgeshared-3.1.3-py3-none-any.whl (21 kB)
  Downloading systembridgeshared-3.1.2-py3-none-any.whl (21 kB)
  Downloading systembridgeshared-3.1.1-py3-none-any.whl (19 kB)
  Downloading systembridgeshared-3.1.0-py3-none-any.whl (19 kB)
  Downloading systembridgeshared-3.0.9-py3-none-any.whl (17 kB)
  Downloading systembridgeshared-3.0.8-py3-none-any.whl (17 kB)
  Downloading systembridgeshared-3.0.7-py3-none-any.whl (17 kB)
Collecting pydantic==1.9.0
  Downloading pydantic-1.9.0-cp37-cp37m-win_amd64.whl (2.0 MB)
     ---------------------------------------- 2.0/2.0 MB 181.4 kB/s eta 0:00:00
Collecting systembridgeshared
  Downloading systembridgeshared-3.0.6-py3-none-any.whl (16 kB)
  Downloading systembridgeshared-3.0.5-py3-none-any.whl (9.5 kB)
  Downloading systembridgeshared-3.0.4-py3-none-any.whl (9.4 kB)
  Downloading systembridgeshared-3.0.3-py3-none-any.whl (9.4 kB)
  Downloading systembridgeshared-3.0.2-py3-none-any.whl (9.3 kB)
  Downloading systembridgeshared-3.0.1-py3-none-any.whl (9.3 kB)
  Downloading systembridgeshared-3.0.0-py3-none-any.whl (9.3 kB)
ERROR: Cannot install systembridgeshared==3.0.0, systembridgeshared==3.0.1, systembridgeshared==3.0.2, systembridgeshared==3.0.3, systembridgeshared==3.0.4, systembridgeshared==3.0.5, systembridgeshared==3.0.6, systembridgeshared==3.0.7, systembridgeshared==3.0.8, systembridgeshared==3.0.9, systembridgeshared==3.1.0, systembridgeshared==3.1.1, systembridgeshared==3.1.2, systembridgeshared==3.1.3, systembridgeshared==3.1.4, systembridgeshared==3.1.5, systembridgeshared==3.2.0, systembridgeshared==3.2.1, systembridgeshared==3.3.0, systembridgeshared==3.3.1, systembridgeshared==3.3.2, systembridgeshared==3.3.3, systembridgeshared==3.3.4, systembridgeshared==3.4.0, systembridgeshared==3.4.1, systembridgeshared==3.4.2, systembridgeshared==3.4.3, systembridgeshared==3.4.4, systembridgeshared==3.4.5, systembridgeshared==3.4.6 and systembridgeshared==3.4.7 because these package versions have conflicting dependencies.

The conflict is caused by:
    systembridgeshared 3.4.7 depends on pandas==1.4.4
    systembridgeshared 3.4.6 depends on pandas==1.4.4
    systembridgeshared 3.4.5 depends on pandas==1.4.4
    systembridgeshared 3.4.4 depends on pandas==1.4.3
    systembridgeshared 3.4.3 depends on pandas==1.4.3
    systembridgeshared 3.4.2 depends on pandas==1.4.3
    systembridgeshared 3.4.1 depends on pandas==1.4.3
    systembridgeshared 3.4.0 depends on pandas==1.4.3
    systembridgeshared 3.3.4 depends on pandas==1.4.3
    systembridgeshared 3.3.3 depends on pandas==1.4.3
    systembridgeshared 3.3.2 depends on pandas==1.4.3
    systembridgeshared 3.3.1 depends on pandas==1.4.3
    systembridgeshared 3.3.0 depends on pandas==1.4.3
    systembridgeshared 3.2.1 depends on pandas==1.4.2
    systembridgeshared 3.2.0 depends on pandas==1.4.2
    systembridgeshared 3.1.5 depends on pandas==1.4.2
    systembridgeshared 3.1.4 depends on pandas==1.4.2
    systembridgeshared 3.1.3 depends on pandas==1.4.2
    systembridgeshared 3.1.2 depends on pandas==1.4.2
    systembridgeshared 3.1.1 depends on pandas==1.4.2
    systembridgeshared 3.1.0 depends on pandas==1.4.2
    systembridgeshared 3.0.9 depends on pandas==1.4.2
    systembridgeshared 3.0.8 depends on pandas==1.4.2
    systembridgeshared 3.0.7 depends on pandas==1.4.2
    systembridgeshared 3.0.6 depends on pandas==1.4.2
    systembridgeshared 3.0.5 depends on pandas==1.4.2
    systembridgeshared 3.0.4 depends on pandas==1.4.2
    systembridgeshared 3.0.3 depends on pandas==1.4.2
    systembridgeshared 3.0.2 depends on pandas==1.4.2
    systembridgeshared 3.0.1 depends on pandas==1.4.2
    systembridgeshared 3.0.0 depends on pandas==1.4.2

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict

ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/topics/dependency-resolution/#dealing-with-dependency-conflicts`

### What Platform / OS are you running?

Windows

### What version are you running?

Latest

### Anything in the logs or a references that might be useful?

```txt
ERROR: Cannot install systembridgeshared==3.0.0, systembridgeshared==3.0.1, systembridgeshared==3.0.2, systembridgeshared==3.0.3, systembridgeshared==3.0.4, systembridgeshared==3.0.5, systembridgeshared==3.0.6, systembridgeshared==3.0.7, systembridgeshared==3.0.8, systembridgeshared==3.0.9, systembridgeshared==3.1.0, systembridgeshared==3.1.1, systembridgeshared==3.1.2, systembridgeshared==3.1.3, systembridgeshared==3.1.4, systembridgeshared==3.1.5, systembridgeshared==3.2.0, systembridgeshared==3.2.1, systembridgeshared==3.3.0, systembridgeshared==3.3.1, systembridgeshared==3.3.2, systembridgeshared==3.3.3, systembridgeshared==3.3.4, systembridgeshared==3.4.0, systembridgeshared==3.4.1, systembridgeshared==3.4.2, systembridgeshared==3.4.3, systembridgeshared==3.4.4, systembridgeshared==3.4.5, systembridgeshared==3.4.6 and systembridgeshared==3.4.7 because these package versions have conflicting dependencies.

The conflict is caused by:
    systembridgeshared 3.4.7 depends on pandas==1.4.4
    systembridgeshared 3.4.6 depends on pandas==1.4.4
    systembridgeshared 3.4.5 depends on pandas==1.4.4
    systembridgeshared 3.4.4 depends on pandas==1.4.3
    systembridgeshared 3.4.3 depends on pandas==1.4.3
    systembridgeshared 3.4.2 depends on pandas==1.4.3
    systembridgeshared 3.4.1 depends on pandas==1.4.3
    systembridgeshared 3.4.0 depends on pandas==1.4.3
    systembridgeshared 3.3.4 depends on pandas==1.4.3
    systembridgeshared 3.3.3 depends on pandas==1.4.3
    systembridgeshared 3.3.2 depends on pandas==1.4.3
    systembridgeshared 3.3.1 depends on pandas==1.4.3
    systembridgeshared 3.3.0 depends on pandas==1.4.3
    systembridgeshared 3.2.1 depends on pandas==1.4.2
    systembridgeshared 3.2.0 depends on pandas==1.4.2
    systembridgeshared 3.1.5 depends on pandas==1.4.2
    systembridgeshared 3.1.4 depends on pandas==1.4.2
    systembridgeshared 3.1.3 depends on pandas==1.4.2
    systembridgeshared 3.1.2 depends on pandas==1.4.2
    systembridgeshared 3.1.1 depends on pandas==1.4.2
    systembridgeshared 3.1.0 depends on pandas==1.4.2
    systembridgeshared 3.0.9 depends on pandas==1.4.2
    systembridgeshared 3.0.8 depends on pandas==1.4.2
    systembridgeshared 3.0.7 depends on pandas==1.4.2
    systembridgeshared 3.0.6 depends on pandas==1.4.2
    systembridgeshared 3.0.5 depends on pandas==1.4.2
    systembridgeshared 3.0.4 depends on pandas==1.4.2
    systembridgeshared 3.0.3 depends on pandas==1.4.2
    systembridgeshared 3.0.2 depends on pandas==1.4.2
    systembridgeshared 3.0.1 depends on pandas==1.4.2
    systembridgeshared 3.0.0 depends on pandas==1.4.2

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict

ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/topics/dependency-resolution/#dealing-with-dependency-conflicts`
```

**Comment by zilpe** [comment]
Did you find a solution to this?

**Comment by timmo001** [comment]
This should be fixed in version [3.5.0](https://github.com/timmo001/system-bridge/releases/latest)
